components/editor/TimesheetEntryEditor.tsx

```diff
@@ -215,6 +214,33 @@ function WeeklyHours({ entries, date }: WeeklyHoursProps): JSX.Element {
     )
 }
 
+interface monthlyHoursProps {
+    entries: TimesheetEntry[]
+    date: Date
+}
+
+function MonthlyHours({ entries, date }: monthlyHoursProps): JSX.Element {
+    const selectedDate = new Date(dateToString(date))
+
+    const firstDay = new Date(selectedDate.getFullYear(), date.getMonth(), 1)
+    const lastDay = new Date(selectedDate.getFullYear(), date.getMonth() + 1, 0)
```

Comment:
I think there is a bug in this: The hours of the last day of the month are not displayed in the sum.

```suggestion
    const lastDay = new Date(selectedDate.getFullYear(), date.getMonth() + 1, 1)
```